Makefile

```diff
@@ -21,18 +22,24 @@ pod-install:
 pod-update: pod-repo-update
 	(cd build && pod update)
 
-bundle-exec-pod-install:	
+bundle-exec-pod-install:
 	(cd build && bundle exec pod install)
 
 open:
 	open build/$(PROJECT_NAME).xcworkspace
 
-test:
+unit-test:
```

Comment:
Can you add a test target which executes both unit-test and functional-test?

Reply:
yes just added that